src/public/services/__tests__/PublicFormAuthService.test.ts

```diff
@@ -28,7 +28,7 @@ describe('PublicFormAuthService', () => {
       expect(MockAxios.get).toHaveBeenCalledWith(
         `${PublicFormAuthService.PUBLIC_FORMS_ENDPOINT}/${MOCK_FORM_ID}/auth/redirect`,
         {
-          params: { isPersistentLogin: false },
+          params: { isPersistentLogin: false, encodedQuery: '' },
```

Comment:
tests can then be simplified to remove the encodedQuery parameter.

should we also add a test to check if the API is called with an expected `encodedQuery`?